pettingzoo/utils/wrappers/multi_episode_env.py

```diff
@@ -13,21 +13,30 @@ class MultiEpisodeEnv(BaseWrapper):
     When there are no more valid agents in the underlying environment, the environment is automatically reset.
     After `num_episodes` have been run internally, the environment terminates normally.
     The result of this wrapper is that the environment is no longer Markovian around the environment reset.
+
+    When `starting_utility` is used, all agents start with a base amount of health points (think of this as poker chips).
```

Comment:
This is an interesting way to do it, but I'm not sure if this is the best name for it. Can't really think of anything else besides `starting_reward` or something. Maybe it could be like `total_rewards` to indicate that it makes the rewards track between resets. Or `tally_rewards`. Feel like `starting_utility` doesn't mean anything to me if I don't know how it works. Would make sense and be simple if it was `total_rewards` adds or subtracts by round.

Reply:
I think utility makes more sense, this is a pretty standard way of saying "starting amount of meaningful substance", starting reward can be confused for the reward given to the agent for starting a new episode, while tally rewards sounds like it should be a boolean.